source/language/types.rst

```diff
@@ -279,25 +279,53 @@ their argument expression in parentheses:
-      | ln     |
-      +--------+
-      | sin    |
-      +--------+
-      | sqrt   |
-      +--------+
-      | tan    |
-      +--------+
+      +----------+--------------------------------+---------------------+
+      | Function | [Input Range/Type] , [...]     | [Output Range/Type] |
+      +==========+================================+=====================+
+      | arccos   | [-1, 1]                        | [0, pi rad]         |
+      +----------+--------------------------------+---------------------+
+      | arcsin   | [-1, 1]                        | [-pi/2, pi/2 rad]   |
+      +----------+--------------------------------+---------------------+
+      | arctan   | [-∞, Real, ∞]                  | [-pi/2, pi/2 rad]   |
+      +----------+--------------------------------+---------------------+
+      | ceiling  | [float], [float]               | [float]             |
+      +----------+--------------------------------+---------------------+
+      | cos      | [float]                        | [float]             |
+      +----------+--------------------------------+---------------------+
+      | div      | ??                             | ??                  |
+      +----------+--------------------------------+---------------------+
+      | exp      | [float]                        | [float]             |
+      |          |                                |                     |
+      |          | [complex]                      | [complex]           |
+      +----------+--------------------------------+---------------------+
+      | floor    | [float], [float]               | [float]             |
+      +----------+--------------------------------+---------------------+
+      | log *(e)*| [float]                        | [float]             |
+      +----------+--------------------------------+---------------------+
+      |  mod     | [int], [int]                   | [int]               |
+      |          |                                |                     |
+      |          | [float], [int,float]           | [float]             |
+      +----------+--------------------------------+---------------------+
+      | popcount | [bits, unsigned]               | [positive int]      |
+      +----------+--------------------------------+---------------------+
+      | pow      | [int], [unsigned int]          | [int]               |
+      |          |                                |                     |
+      |          | [float], [int,float]           | [float]             |
+      |          |                                |                     |
+      |          | [complex], [int,float,complex] | [complex]           |
+      +----------+--------------------------------+---------------------+
+      | rotl     | ??                             | ??                  |
+      +----------+--------------------------------+---------------------+
+      | rotr     | ??                             | ??                  |
```

Comment:
These presumably take in `bit`-likes (`bit` and potentially `uint` depending on how the type system shakes out) and theoretically they could take an `int` or `uint`.  I'd probably expect as a user that `rotl(a, b) == rotr(a, -b)`, and I can't immediately think of any problems with that.

Reply:
Thanks

Reply:
Done.